pymisp/mispevent.py

```diff
@@ -109,18 +110,12 @@ def __init__(self, describe_types=None, strict=False):
             :strict: If false, fallback to sane defaults for the attribute type if the ones passed by the user are incorrect
         """
         super(MISPAttribute, self).__init__()
-        if not describe_types:
-            ressources_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'data')
-            with open(os.path.join(ressources_path, 'describeTypes.json'), 'rb') as f:
-                if OLD_PY3:
-                    t = json.loads(f.read().decode())
-                else:
-                    t = json.load(f)
-            describe_types = t['result']
-        self.__categories = describe_types['categories']
-        self._types = describe_types['types']
```

Comment:
Would it make sense to use double-underscores here as well?

Reply:
Absolutely.